### DISCUSSION TITLE
Buildings contour too thick (default theme)

### DISCUSSION THREAD

**Discussion by andrixnet** [mapsforge/vtm#1142] [Q&A] [answered]
I've been directed by the c:geo team to report a problem here as it might be upstream mapsforge (they said).
Please look at issue https://github.com/cgeo/cgeo/issues/16096 in the c:geo project. 

Using what c:geo calls "unified map" (developers also called it VTM), at certain zoom levels building contours are rendered as excessively thick, resulting in hard to read map. See screenshots in the issue.

Thank you.

**Comment by devemux86** [comment, marked as answer]
@andrixnet Thanks for the report.

As we can not know how each app uses the libraries,
the reports must be reproduced with the Samples of the libraries.

I already answered in the topic that using the Samples we can not see thick building lines:
- https://github.com/cgeo/cgeo/issues/16096#issuecomment-2365164785

Also, OpenAndroMaps themes are created with Mapsforge in mind,
so better use them with the Mapsforge engine.

Sample apps of Mapsforge vs VTM:

<img src="https://github.com/user-attachments/assets/53bbcce1-29e7-4af6-ac10-72877c413570" width="50%"/><img src="https://github.com/user-attachments/assets/d03f677c-0392-4025-ba2a-3040c2323924" width="50%"/>

**Reply by andrixnet** [reply]
I understand the differences. Also, OpenAndroMaps are not involved in the comparison. 
In the comment mentioned, one does see slightly thicker contour lines for buildings on the right side image, though not as much of a difference as in my report (as end-user). 
Would have been interesting to see another comparison with one zoom level out and building contours.
